### DISCUSSION TITLE
AlDente 1.22 "Fettuccine"

### DISCUSSION THREAD

**Discussion by davidwernhart** [AppHouseKitchen/AlDente-Battery_Care_and_Monitoring#969] [General]
- [Pro] [Thunderbolt 3 Macbooks] Exciting new Beta Feature: Power Flow. Check it out here: [Feature Explanation: Power Flow](https://apphousekitchen.com/feature-explanation-power-flow/).

- [Pro] Completely redesigned Menubar icons to better blend into macOS.

- [Pro] Added the option to use the battery icon of macOS, optional in colors.

- [Pro] [Non-Thunderbolt 3 Macbooks] When AlDente is closed or the MacBook is turned off, the persistent charge limit is now set to the lower limit of the set Sailing Mode range.

- [Pro] Settings now get backed up when a pro license is deactivated and can be restored once the Pro version is activated again.

- [Pro] Fixed a bug that caused the Sailing Mode UI to show wrong numbers while Calibration Mode is active.

- [Pro] Fixed a bug that caused Calibration Mode to cancel immediately when triggered through a Schedule task while the power cable is not connected.

- [All] Refreshed the UI design of the popup window.

- [All] Fixed a bug that caused faulty UI layout generation for some languages.

- [All] Fixed a bug where AlDente did not register when the helper removal is canceled properly.

- [All] Added a warning message when the discharge button is clicked while the charge limit is higher than the current percentage.

- [All] Added a "Don't show again" checkbox to the helper installation popup.

- [All] Fixed a bug that caused the app to hang after a successful helper installation in some scenarios.

With peace & love from Vienna,  
AppHouseKitchen

<hr /><em>This discussion was created from the release <a href='https://github.com/davidwernhart/AlDente-Charge-Limiter/releases/tag/1.22'>AlDente 1.22 "Fettuccine"</a>.</em>

**Comment by Snipeye** [comment]
Love it!  As a request/suggestion (from a pro license owner) with the "use native battery icon" feature, this totally replaces the battery menu bar icon for me - except that clicking on the built-in/native macos "battery" icon is a shortcut to seeing energy-heavy apps.  Can you add a toggle in to enable a widget for that under your menu?  Alternatively, could the Power Flow Sankey show individual apps rather than just the "macbook"?

Thanks for all the hard work!

**Reply by MatthiasKerbl** [reply]
Hi @Snipeye,

Thank you for the ideas. This might be something we could add in the future.

Best,
Matthias

**Comment by adrianedelcu** [comment]
I just logged in into my Github account to congratulate you guys and say thank you for this new update, specially since I have been seeking a decent menubar icon to blend with the macbook since I became a Pro user. I love the new Power Flow feature.

**Comment by alfimnaushad** [comment]
After the update sailing mode doesn't seem to work. Is anybody else experiencing the same issue?

**Reply by DamonHD** [reply]
Seems still to be working for me (M1 MBA, Ventura 13.4.1).

**Reply by putrudius** [reply]
I have the same issue with sail mode no longer working in version 1.22 on two M1 Pro Macs, one Ventura and the other Monterey.

I downgraded to 1.21.4, (including removing and reinstalling the helper app) and sail mode is working again.

I've reported the issue via support.

There is a way to generate and submit a debug report and I'll try doing this with the other Mac that still has 1.22 running.  The instructions for this is here: https://apphousekitchen.com/how-to-generate-and-share-a-debug-file/
